From 7bdcdbc184082703435d3f6b8ce9baf4176cbc6c Mon Sep 17 00:00:00 2001
From: Han Xu <han.xu@nxp.com>
Date: Thu, 28 Sep 2017 09:59:51 -0500
Subject: [PATCH 2637/5242] MLK-16572: mtd: gpmi-nand: change the NAND omux
 setting

commit  9d6f601d2857444c9cf6cab52e949d82af2b7f8d from
https://source.codeaurora.org/external/imx/linux-imx.git

use the dqs and re pins rather than dqs_n/dqs_p re_n/re_p pins for NAND

BuildInfo: SCFW 9e9f6ec6, IMX-MKIMAGE 0, ATF 0

Reviewed-by: Frank Li <frank.li@nxp.com>
Signed-off-by: Han Xu <han.xu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../fsl-imx8qxp-lpddr4-arm2-gpmi-nand.dts          |    8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2-gpmi-nand.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2-gpmi-nand.dts
index 774a1da..b85c69d 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2-gpmi-nand.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2-gpmi-nand.dts
@@ -40,10 +40,10 @@
 				SC_P_USDHC1_DATA2_CONN_NAND_WE_B	0x0e00004c
 				SC_P_USDHC1_DATA3_CONN_NAND_ALE		0x0e00004c
 				SC_P_USDHC1_CMD_CONN_NAND_CE0_B		0x0e00004c
-				SC_P_USDHC1_VSELECT_CONN_NAND_RE_P	0x0e00004c
-				SC_P_USDHC1_WP_CONN_NAND_DQS_N		0x0e00004c
-				SC_P_USDHC1_CD_B_CONN_NAND_DQS_P	0x0e00004c
-				SC_P_USDHC1_RESET_B_CONN_NAND_RE_N	0x0e00004c
+
+				/* i.MX8QXP NAND use nand_re_dqs_pins */
+				SC_P_EMMC0_CMD_CONN_NAND_DQS		0x0e00004c
+				SC_P_USDHC1_DATA1_CONN_NAND_RE_B	0x0e00004c
 			>;
 		};
 	};
-- 
1.7.9.5

